src/main/java/cc/hyperium/integrations/betterchatfilter/BetterChatFilter.java

```diff
@@ -59,11 +59,7 @@ public BetterChatFilter() {
     } catch (Exception e) {
       e.printStackTrace();
       // After failing to download file, make arraylist to make up for it.  
-      ArrayList rawBadwords = new ArrayList<String>();
-      rawBadwords.add("fuck");
-      rawBadwords.add("shit"); 
-      rawBadwords.add("damn");
-      rawBadwords.add("bitch");
+      String rawBadwords = "fuck shit bitch damn";
       badWords = new ArrayList<>(Arrays.asList(rawBadwords.split("\n")));
```

Comment:
Use Arrays.asList with the words instead
ex:
`Arrays.asList("word1", "word2", ...)`

Also "rawBadwords" doesn't contain line separator and you're trying to split using the line separator